baselines/dom.generated.d.ts

```diff
@@ -10805,30 +10817,21 @@ declare var NavigationPreloadManager: {
 };
 
 /** The state and the identity of the user agent. It allows scripts to query it and to register themselves to carry on some activities. */
-interface Navigator extends NavigatorID, NavigatorOnLine, NavigatorContentUtils, NavigatorStorageUtils, MSNavigatorDoNotTrack, MSFileSaver, NavigatorBeacon, NavigatorConcurrentHardware, NavigatorUserMedia, NavigatorLanguage, NavigatorStorage, NavigatorAutomationInformation {
-    readonly activeVRDisplays: ReadonlyArray<VRDisplay>;
```

Comment:
We should restore `activeVRDisplays`/`getVRDisplays()` and `NavigatorBeacon`. Looks good otherwise.